modules/ossm-threescale-webassembly-module-minimal-working-configuration.adoc

```diff
@@ -14,8 +14,9 @@ kind: ServiceMeshExtension
 metadata:
   name: threescale-auth
 spec:
-  image: registry.redhat.io/openshift-service-mesh/3scale-auth-wasm-rhel8:0.0.1
-  phase: PostAuthZ
+  image: registry.redhat.io/3scale-amp2/3scale-auth-wasm-rhel8:0.0.3
+  imagePullSecret: optional-pull-secret-resource
```

Comment:
Is `optional-pull-secret-resource` the verbatim name of the `imagePullSecret`, or is this a value the user can replace with their own data as appropriate? 

Really, I have this same question throughout this file. Values that a user would replace with their own from their environment/configuration should be formatted as e.g. `<user_key>`. I am not 100% sure that is what is being represented in this YAML sample, but if so, it should be updated to match that format. 

For details, see the bullet that starts "To mark up command syntax, use the code block and wrap any replaceable values in angle brackets" in the [OCP docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) (sorry I can't direct link to the bullet).